Cache/IO.lean

```diff
@@ -44,6 +44,20 @@ def LIBDIR : FilePath :=
 def IRDIR : FilePath :=
   ".lake" / "build" / "ir"
 
+/--
+TODO: is there a better test to see if a module is part of Lean core?
+-/
+def isInLeanCore (mod : Name) :=
+  #[`Init, `Lean, `Std, `Lake].contains mod.getRoot
+
+/--
+TODO: write a better test which modules are part of the mathlib cache
+-/
+def isPartOfMathlibCache (mod : Name) :=
+  #[`Mathlib, `Batteries, `Aesop, `Cli, `ImportGraph,
+    `LeanSearchClient, `Plausible, `Qq, `ProofWidgets,
+    `Archive, `Counterexamples, `MathlibTest].contains mod.getRoot
```

Comment:
I think this would be better with one entry per line, for the sake of diffing / downstream patching